Header header

#The name of this gripper target
string name

#The pose of this gripper target
geometry_msgs/Pose pose

# F/T thresholds to use for pickup and place
float64[] pickup_ft_threshold
float64[] place_ft_threshold
